Sort explicit import lists

jship opened the pull request:
Sort explicit import lists - starting capitals first (i.e. typeclasses/types), then operators, then identifiers.

sighingnow commented on TESTS.md:
````diff
@@ -64,6 +64,93 @@ import A
+```haskell
+import Control.Monad.Trans.Control
+       (MonadBaseControl(restoreM, liftBaseWith))
+```
````
Why explicitly imported methods are not sorted ?

sighingnow commented on TESTS.md:
````diff
@@ -64,6 +64,93 @@ import A
+import qualified MegaModule as M
+       (Either, Maybe(Nothing, Just), MaybeT(..), Monad, MonadBaseControl,
+        (<<<), (>>>), join, liftIO, void)
+```
````
I think the big example is enough to demonstrate this feature. Describing the rules together before or after the big example source code would be better the catch the idea of the sorting rules.

jship commented:
@sighingnow: Thanks for the feedback.

Added support for sorting explicitly imported data constructors & methods.  Cleaned up `Tests.md` to just have the big example.
